audiences-react/src/ujs.js

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import { AudienceEditor } from "."
+
+if (typeof window !== "undefined") {
+  window.AudiencesRails = { AudienceEditor }
+}
+
+function mountAudiencesEditors() {
+  document
+    .querySelectorAll("[data-react-class='AudiencesEditor']")
+    .forEach((element) => {
+      if (!element.dataset.reactMounted) {
+        const uri = element.dataset.audiencesUri
+        const context = element.dataset.audiencesContext
```

Comment:
Let's make sure `element.dataset.reactMounted` is set to true somewhere, so the element is not overridden on an unrelated event handling:

```suggestion
        const context = element.dataset.audiencesContext
        element.dataset.reactMounted = true
```